megatron/rl/inference/megatron.py

```diff
@@ -144,6 +157,13 @@ async def base_generate(self, request: InferenceRequest):
 
         tokenizer = get_tokenizer()
 
+        if tokenizer.bos is None:
```

Comment:
This is very spammy. I notice when training with a tokenizer without BOS this log repeats hundreds of times.

Reply:
Addressed!